tickets/src/test/java/no/unit/nva/publication/ticket/create/CreateTicketHandlerTest.java

```diff
@@ -699,17 +673,102 @@ void shouldAllowCuratorWithValidAccessRightToCreateTicket(PublicationStatus stat
+        var requestBody = constructDto(ticketType);
+
+        var curatingInstitution = publication.getCuratingInstitutions().iterator().next();
+        var request = createHttpTicketCreationRequest(requestBody, publication.getIdentifier(),
+                                                      curatingInstitution, randomUri(), randomString(),accessRight);
+        handler.handleRequest(request, output, CONTEXT);
+
+        var response = GatewayResponse.fromOutputStream(output, Void.class);
+        assertThat(response.getStatusCode(), is(equalTo(HTTP_CREATED)));
+    }
+
+    @ParameterizedTest
+    @MethodSource("ticketEntryProvider")
+    void shouldNotAllowContributorToCreateTicketForDegreePublication(Class<? extends TicketEntry> ticketType)
+        throws ApiGatewayException, IOException {
+        var publication = TicketTestUtils.createPersistedDegreePublication(PUBLISHED, resourceService);
+        var requestBody = constructDto(ticketType);
+
+        var curatingInstitution = publication.getCuratingInstitutions().iterator().next();
+        var contributorCristinId = publication.getEntityDescription().getContributors().getFirst().getIdentity().getId();
+        var request = createHttpTicketCreationRequest(requestBody, publication.getIdentifier(),
+                                                      curatingInstitution, contributorCristinId, randomString());
+        handler.handleRequest(request, output, CONTEXT);
+
+        var response = GatewayResponse.fromOutputStream(output, Void.class);
+        assertThat(response.getStatusCode(), is(equalTo(HTTP_FORBIDDEN)));
+    }
+
+    @ParameterizedTest
+    @MethodSource("no.unit.nva.publication.ticket.test.TicketTestUtils#ticketTypeAndAccessRightProvider")
+    void shouldNotAllowCuratorWithValidAccessRightAndRelatedContributorToCreateTicketForDegreePublication(
+        PublicationStatus publicationStatus, Class<? extends TicketEntry> ticketType, AccessRight accessRight)
+        throws ApiGatewayException, IOException {
+        var publication = TicketTestUtils.createPersistedDegreePublication(publicationStatus, resourceService);
+        var requestBody = constructDto(ticketType);
+
+        var curatingInstitution = publication.getCuratingInstitutions().iterator().next();
+        var request = createHttpTicketCreationRequest(requestBody, publication.getIdentifier(),
+                                                      curatingInstitution, randomUri(), randomString(),accessRight);
+        handler.handleRequest(request, output, CONTEXT);
+
+        var response = GatewayResponse.fromOutputStream(output, Void.class);
+        assertThat(response.getStatusCode(), is(equalTo(HTTP_FORBIDDEN)));
+    }
+
+    @ParameterizedTest
+    @MethodSource("no.unit.nva.publication.ticket.test.TicketTestUtils#ticketTypeAndAccessRightProvider")
+    void shouldNotAllowNotRelatedCuratorWithValidAccessRightForNonDegreePublication(
```

Comment:
Editor is covered elsewhere?

Reply:
Editor is covered in EditorPermissionStrategy.